src/screens/SiteDetailScreen.js

```diff
@@ -145,6 +145,7 @@ class SiteDetailScreen extends Component {
                     </Text>
                     <TouchableOpacity
                         onPress={() => Expo.WebBrowser.openBrowserAsync(mvum_links[selectedSite.mvum])}
+                        // onPress={() => navigate(navKeys.MVUM_INSPECTOR)}
```

Comment:
GH-104 -- Once we figure out how to render local pdfs, we'll navigate to the new screen instead of opening a web browser.